packages/babel-compat-data/scripts/utils-build-data.js

```diff
@@ -35,6 +32,13 @@ exports.environments = [
   "samsung",
 ];
 
+function flatMap(array, cb) {
+  if (array.flatMap) return array.flatMap(cb);
+  return array.reduce((flattened, value) => {
+    return flattened.concat(value);
+  }, []);
+}
```

Comment:
We could just bump the Node.js version requirement for building Babel to 12 (currently it's 10 I think), and avoid the polyfill.

Reply:
that will be a breaking change, wdyt about doing this for babel8?

Reply:
It's not a breaking change, it's just the minimum Node.js versions used when _building_ Babel (it doesn't affect the code published on npm, which is subject to semver).

Reply:
Actually, the minimum Node.js version is already 12 since many scripts we use are native ES modules. We can remove the polyfill without problems.

Reply:
That's good, because I forgot to call `cb` in this polyfill….. 😅